cardano-cli/src/Cardano/CLI/EraBased/Run/Query.hs

```diff
@@ -970,17 +971,30 @@ callQueryStakeAddressInfoCmd
         (stakeRewardAccountBalances, stakePools) <-
           easyRunQuery (queryStakeAddresses sbe stakeAddr networkId)
 
-        beo <- requireEon BabbageEra era
+        ceo <- requireEon ConwayEra era
 
-        stakeDelegDeposits <- easyRunQuery (queryStakeDelegDeposits beo stakeAddr)
+        stakeVoteDelegatees <- easyRunQuery (queryStakeVoteDelegatees ceo stakeAddr)
+        stakeDelegDeposits <- easyRunQuery (queryStakeDelegDeposits (convert ceo) stakeAddr)
 
-        stakeVoteDelegatees <- monoidForEraInEonA era $ \ceo ->
-          easyRunQuery (queryStakeVoteDelegatees ceo stakeAddr)
+        govActionStates :: (Seq.Seq (L.GovActionState (ShelleyLedgerEra era))) <-
```

Comment:
This field could be made optional and only show up in eras > Conway. This way the query would still be usable in Babbage.

Reply:
With the latest chantes the query still works in Babbage era:

cardano-cli babbage query stake-address-info --address $(< example/utxo-keys/stake1.addr)
WARNING: Selected era is deprecated and will be removed in the future.
[
    {
        "address": "stake_test1uq07mnmf838hdlu9qpnwjnntrrtvsa6448ud0g32qkm0kecusq4sd",
        "delegation": "pool1xj26f4rqk2t049yu6n4mmg5zstr6sarv63nc5fuw60rx7u0jxef",
        "delegationDeposit": 2000000,
        "rewardAccountBalance": 0
    }
]